### PR TITLE
ITS-3306 Separate Service functions from CheckDetailsController to a new  component.

### PR DESCRIPTION
**Pull request by rambabu-posa**

### CODE REVIEW

File: test/xx.scala [hunk 1]
```diff
@@ -0,0 +1,503 @@
+/*
```
Comment (ajchilts) [inline test/xx.scala:1]: Do we need this file? Looks to be all commented out?

**Review by ajchilts — CHANGES_REQUESTED** [review]
A good start! There might be more we can do to refactor out the complexity and reduce the function length. Let's maybe pair up tomorrow and take a look?

File: test/xx.scala (hunk 1 above)
Comment (rambabu-posa) [reply test/xx.scala:1]: oops, That was my local reference and pushed without my knowledge. will remove it. thanks

**Review by ajchilts — DISMISSED** [review]

**Review by ajchilts — APPROVED** [review]
LGTM

**Pull request merged**